ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -332,29 +328,28 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
-		888DAA1A29B48C8900311304 /* Frameworks */ = {
+		6A774A4C9B67562633967EEE /* Frameworks */ = {
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				E759201CFC6E9949F71D1B5A /* libPods-BetterSocial-BetterSocialStaging.a in Frameworks */,
+				F23408F1E96AC908FB2D1D88 /* libPods-OneSignalNotificationServiceExtension.a in Frameworks */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
-		889CAB6427D46CD6007F5892 /* Frameworks */ = {
+		888DAA1A29B48C8900311304 /* Frameworks */ = {
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				92D8142362E828E0BFD1A9D0 /* BuildFile in Frameworks */,
-				93983F146E51F87F73BDAD47 /* libPods-BetterSocial-BetterSocialDev.a in Frameworks */,
+				E759201CFC6E9949F71D1B5A /* libPods-BetterSocial-BetterSocialStaging.a in Frameworks */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
-		C4AE08FF2AC43A4800E13758 /* Frameworks */ = {
+		889CAB6427D46CD6007F5892 /* Frameworks */ = {
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				962349FAE6BEC6C5CCF1D8EB /* BuildFile in Frameworks */,
-				1DBF80C9D3A1696FFC841AE6 /* libPods-OneSignalNotificationServiceExtension.a in Frameworks */,
+				92D8142362E828E0BFD1A9D0 /* BuildFile in Frameworks */,
+				93983F146E51F87F73BDAD47 /* libPods-BetterSocial-BetterSocialDev.a in Frameworks */,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here seem to be related to the reordering of build phases and removal of some files from the Frameworks build phase. As long as these changes are intentional and the removed files are no longer necessary, this should be fine.



<!-- This is an auto-generated comment by OSS CodeRabbit -->